### PR TITLE
Benchmarks: micro benchmarks - add int8 support for cublaslt function

### PR DESCRIPTION
**Pull request by yukirora**
**Description**
add int8 support for cublaslt function.

### CODE REVIEW

**Review by cp5555 — APPROVED** [review]

File: superbench/benchmarks/micro_benchmarks/cublaslt_gemm/cublaslt_gemm.cu [hunk 1]
```diff
@@ -16,6 +16,7 @@ using fp16 = half;
 using bf16 = nv_bfloat16;
 using fp8e4m3 = __nv_fp8_e4m3;
 using fp8e5m2 = __nv_fp8_e5m2;
+using int8= int8_t;
```
Comment (abuccts) [inline superbench/benchmarks/micro_benchmarks/cublaslt_gemm/cublaslt_gemm.cu:19]: format

File: superbench/benchmarks/micro_benchmarks/cublaslt_gemm/cublaslt_utils.cc [hunk 2]
```diff
@@ -62,6 +62,8 @@ void cublasLtGemm::Setup(int m, int n, int k, int batch, int lda, int ldb, int l
         gemm_compute_type = CUBLAS_COMPUTE_32F;
     if (a_type == CUDA_R_64F || b_type == CUDA_R_64F)
         gemm_compute_type = CUBLAS_COMPUTE_64F;
+    if (a_type==CUDA_R_8I)
```
Comment (abuccts) [inline superbench/benchmarks/micro_benchmarks/cublaslt_gemm/cublaslt_utils.cc:65]: format

**Review by abuccts — APPROVED** [review]

File: superbench/benchmarks/micro_benchmarks/cublaslt_gemm/cublaslt_gemm.cu [hunk 3]
```diff
@@ -84,6 +85,8 @@ template <typename T> cudaDataType_t get_datatype() {
         return CUDA_R_8F_E4M3;
     if (std::is_same<T, fp8e5m2>::value)
         return CUDA_R_8F_E5M2;
+    if (std:: is_same<T, int8>::value)
```
Comment (abuccts) [inline superbench/benchmarks/micro_benchmarks/cublaslt_gemm/cublaslt_gemm.cu:88]: format

File: tests/benchmarks/micro_benchmarks/test_cublaslt_function.py [hunk 4]
```diff
@@ -63,15 +63,15 @@ def test_cublaslt_gemm_command_generation(self):
         (benchmark_cls, _) = BenchmarkRegistry._BenchmarkRegistry__select_benchmark(self.benchmark_name, Platform.CUDA)
         benchmark = benchmark_cls(
             self.benchmark_name,
-            parameters='--batch 2:16:2 --shapes 2:4,4:8,8:32 32:128:4,128,128 --in_types fp16 fp32 fp64',
+            parameters='--batch 2:16:2 --shapes 2:4,4:8,8:32 32:128:4,128,128 --in_types fp16 fp32 fp64 int8',
         )
         self.assertTrue(benchmark._preprocess())
         self.assertEqual(4 * (2 * 2 * 3 + 2) * 3, len(benchmark._commands))
```
Comment (abuccts) [inline tests/benchmarks/micro_benchmarks/test_cublaslt_function.py:69]: do you need to update last 3 to 4?

Comment (yukirora) [reply tests/benchmarks/micro_benchmarks/test_cublaslt_function.py]: thanks a lot ! updated to use len(benchmarks._args.in_types)

**Pull request merged**